<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkWindow</class>
 <widget class="QWidget" name="WorkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="submitRequest">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交充电请求</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeMode">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>530</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改充电模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeCapacity">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>530</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改充电量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewQueueNum">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看排队号码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewWaitNum">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>205</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看前车等待数量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewDetail">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>355</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看充电详单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endORpay">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结束充电</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitChange">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="modeLab">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>480</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>充电模式:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modeBox">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>480</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>slow</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>slow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fast</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="capacityLab">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>480</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>充电量:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="capacityEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="usrLab">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光隶书_CNKI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="usrText">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光隶书_CNKI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Unknown</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>530</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>华光隶书_CNKI</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>欢迎使用智能充电服务系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="batteryLab">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>480</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电池容量:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="batteryEdit">
   <property name="geometry">
    <rect>
     <x>669</x>
     <y>480</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="infoBrowser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>500</width>
     <height>330</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
font: 10pt &quot;Times New Roman&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="carLab">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光隶书_CNKI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>车辆号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="carButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>↑切换车辆↑</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>80</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
